stutpip123/A3-Antistasi
# optre templates

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added OPTRE templates, cargo and logistics

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

These templates use an experimental trait to change base class, so it needs to be tested if the AI still works correctly (I've done this on LAN, should work on dedicated)

### How can the changes be tested?
Steps:
Download this mod: Will release an experimental version soon. Needed to make the base class trait work correctly, else you'll get flooded with errors

Play a game of antistasi on one of the UNSC or Covenant templates, check if all units work correctly. I have done a small bit of testing but not a full game worth of testing.

********************************************************
Notes:
